### DISCUSSION TITLE
Default page for folder

### DISCUSSION THREAD

**Discussion by e4rthdog** [requarks/wiki#6626] [Help / Questions]
My current content has every article on a separate folder. This folder includes the .md file alongside with a directory for resources (images e.t.c) for this article.

In that way i have  the article standalone.

For example my folder structure is:

```it -> operations -> how-to-login -> index.md```

I would like on the side-tree navigation, when i click on the ```how-to-login``` folder to automatically display the index.md file.

I tried but it doesn't work. I need to click on the index page in the folder.

MKDOCS has this functionality as **Section Index Pages**  

Also Docusaurus has it.

**Comment by NGPixel** [comment]
You need to name your pages the same as your folder, e.g.:

`it/operations/how-to-login.md`

**Comment by e4rthdog** [comment]
I am doing something wrong as it doesn;t work.

My folder structure is:
![image](https://github.com/requarks/wiki/assets/1651764/48152609-ad01-415d-a1a7-3672ad3a950a)

And the wiki representation is:
![image](https://github.com/requarks/wiki/assets/1651764/872a52cc-6e6b-4699-b22c-2effe64fc30c)

Something is off here. Notice that the dot is missing in the menu entry.

I really hope to get this working.

Thanks.

**Reply by NGPixel** [reply]
Read my post correctly. You've put the file inside the folder of the same name. It has to be at the same level as the folder.

**Reply by e4rthdog** [reply]
I did that also. Take a look:
![image](https://github.com/requarks/wiki/assets/1651764/9a4be0c8-ecb7-4974-ba17-fd46991b9f0b)
